web/templates/popovers/left_sidebar_recent_view_popover.hbs

```diff
@@ -2,7 +2,7 @@
     <li>
         <a tabindex="0" class="set-default-view" data-view-code="{{view_code}}">
             <i class="fa fa-home" aria-hidden="true"></i>
-            {{t "Make recent conversations my home view" }}
+            {{> make_my_home_view view_name="recent conversations"}}
```

Comment:
"recent conversations" needs to be translatable. I think this means you should just inline the `make_my_home_view` template in each place.

Reply:
Done.